omero/sysadmins/windows/server-installation.txt

```diff
@@ -146,6 +128,40 @@ The following are optional depending on what services you require:
 
 .. [1] May already have been installed as a dependency of Matplot Lib.
 
+Ice
+^^^
+
+.. note::
+
+    OMERO |version| supports Ice 3.5 and 3.4; support for 3.6 is
+    currently upcoming. You will need to pick the matching server
+    download for the version of Ice you have installed.
+
+    If you are using precompiled packages of Ice and Python
+    (recommended) you should use the versions specified below. In
+    particular since OMERO does not support Python 3 **it will not
+    work with the ZeroC Ice 3.5 package for Windows**. However, Ice
+    3.5 will work when built against Python 2.7.  However, the ZeroC
+    Ice 3.6 packages do support Python 2.7.
+
+Ice 3.5.1 built against Python 2.7 is available from our `Ice
+downloads page <http://downloads.openmicroscopy.org/ice/3.5.1/>`_, or
+Ice 3.4 installers can be found on the old :zeroc:`ZeroC Ice 3.4.2
+download page <download_3_4_2.html>`.  Alternatively, the :zeroc:`Ice
+3.6.0 download page <download.html>` provides installers for Windows,
+but support for this release is currently upcoming.
```

Comment:
Unsure.  Given the level of support and testing, it might be best to leave it public but without any references, so anyone who explicitly asks can be pointed to them, but no one will use them by accident.